pkg/resolver/search.go

```diff
@@ -334,6 +342,13 @@ func getWhereClauseExpression(prop, operator string, values []string) []exp.Expr
 		}
 	case "=":
 		exps = append(exps, goqu.L(`"data"->>?`, prop).In(values))
+	case "@>":
+		for _, val := range values {
+			exps = append(exps, jsonLookup(goqu.L(`"data"->?`, prop), val))
+		}
+	case "?|":
+		exps = append(exps, jsonAnyKeyArrayLookup(goqu.L(`"data"->?`, prop), values))
```

Comment:
Would this be the same? (Sorry, it may reduce coverage 😢 )
```suggestion
	case "@>":
		for _, val := range values {
			exps = append(exps, goqu.L(`"data"->'?' @> '?'` , prop, val))
		}
	case "?|":
		exps = append(exps, goqu.L(`"data"->'?' ? ?`, prop, "?|", values))
```

**Note:** I need to double check the usage of `?`.  I thought postgres uses `$1, $2, ...`